ocr.py

```diff
@@ -44,3 +49,116 @@ def ocr(self, image_file, show_steps=False):
         features = self.feature_extractor.extract(image_file.image, segments)
         classes = self.classifier.classify(features)
         return classes, segments
+
+
+class SimpleOCR(object):
+    def __init__(self, segmentation="contour", extraction="simple", classification="knn",
```

Comment:
In general, I feel this could actually be a function that builds a instance of `OCR`. Most of the methods either would be useful in the `OCR` class itself, or duplicate functionality from there. What do you think?